firmware/application/apps/ui_freqman.hpp

```diff
@@ -157,6 +155,14 @@ class FrequencyManagerView : public FreqManBaseView {
         Color::red(),
         true};
 
+    Button button_edit_entry{
+        {0 * 8, 14 * 16 - 4, 15 * 8, 1 * 16 + 4},
+        "Edit"};
+
+    Rectangle rect_padding{
+        {15 * 8, 14 * 16 - 4, 15 * 8, 1 * 16 + 4},
+        Color::grey()};
+
     Button button_edit_freq{
         {0 * 8, 15 * 16, 15 * 8, 2 * 16},
         "Frequency"};
```

Comment:
I love your new edit screen.  But it's hard to press Edit since the Frequency button below it is so big.  IMO, you should deprecate the old Frequency & Description buttons:  (1) the duplicate functionality is unnecessary, (2) it's hard to press Edit without hitting Frequency instead, (3) the old Frequency edit screen is so confusing with its Save button that opens yet another frequency editing screen (the only redeeming grace is that it has a Load button that allows a frequency to be copied from another file, but even then it loses the description and everything else that went along with the frequency; you could probably do better by moving the Load button to your Edit screen).  Anyways, maybe Phase II?

Reply:
Ooh that give me an idea, we could also maybe use this view for the Save screen.

Reply:
Oddly when I press the bottom half of the "Edit" button it opens up the "Frequency" screen.  Is it my touchscreen or do the fields overlap?

Reply:
Nevermind that comment; my touchscreen apparently needed calibration...  I'm able to select it now consistently.

Reply:
I normally just use the dpad buttons. Select or Right on an entry focuses the Edit button and then use Select to open the edit screen.

Reply:
> Ooh that give me an idea, we could also maybe use this view for the Save screen.

I like that idea! (that is, if you keep the Frequency button at all)   That should save a few ROM bytes too.

Reply:
The flow to get into that save screen is normally from a FrequencyField (e.g. Selecting the freq in Pocsag) and then saving from there. It might be nice to be able to set the other properties.